### PR TITLE
Style terms and conditions

### PR DESCRIPTION
**Pull request by jleach**
# Summary of Changes

Add styling and bring Theme through the terms and conditions screen.

# Related Issues

n/a

# Pull Request Checklist

This is just a reminder about the most common mistakes. Please make sure that you tick all _appropriate_ boxes. But please read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!

If an item doesn't apply to your pull request, **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this).
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components.
- [x] Run prettier: `npm run style-format`
- [x] Updated **documentation** for changed code and new or modified features.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: App/components/buttons/Button.tsx [hunk 1]
```diff
@@ -1,53 +1,74 @@
...
-    alignItems: 'center',
-    padding: 10,
-    marginVertical: 10,
-  },
   disabled: {
     backgroundColor: Colors.shadow,
   },
   neutral: {
-    backgroundColor: Colors.text,
+    backgroundColor: Colors.textColor,
```
Comment (amanji) [inline App/components/buttons/Button.tsx:28]: What's the main difference between `text` and `textColor`? We probably want to stick with one across the app.

Comment (jleach) [reply App/components/buttons/Button.tsx:28]: Fixed.

File: App/Theme.ts [hunk 2]
```diff
@@ -32,22 +56,98 @@ export const Colors: ColorTheme = {
   toastSuccess: BaseColors.green,
   toastError: BaseColors.red,
   toastInfo: BaseColors.black,
+  accent: BaseColors.yellow,
+  borderLight: '#2E8540',
```
Comment (amanji) [inline App/Theme.ts]: Can we add the hex codes to `BaseColors` and reference them here?

File: App/components/buttons/Button.tsx [hunk 3]
```diff
@@ -1,53 +1,74 @@
...
 
   return (
     <TouchableOpacity
       onPress={onPress}
       accessible={accessible}
       accessibilityLabel={accessibilityLabel}
-      style={[styles.button, disabled && styles.disabled, neutral && styles.neutral, negative && styles.negative]}
+      style={[
+        myButtonType === ButtonType.Primary ? Buttons.primary : Buttons.secondary,
+        disabled && (myButtonType === ButtonType.Primary ? Buttons.primaryDisabled : Buttons.secondaryDisabled),
```
Comment (amanji) [inline App/components/buttons/Button.tsx:55]: I've noticed that the disabled styles are overwritten by applying the neutral or negative props. Should probably have disabled get applied last.

File: App/components/buttons/Button.tsx [hunk 4]
```diff
@@ -1,53 +1,74 @@
...
+        negative && styles.negative,
+      ]}
       disabled={disabled}
+      activeOpacity={ActiveOpacity}
     >
-      <Text style={[styles.text, neutral && { color: Colors.shadow }]}>{title}</Text>
+      <Text
+        style={[
+          myButtonType === ButtonType.Primary ? Buttons.primaryText : Buttons.secondaryText,
+          disabled &&
```
Comment (amanji) [inline App/components/buttons/Button.tsx:65]: Same comment as above for disabled styling

File: App/components/texts/TextBox.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    backgroundColor: Colors.background,
+  },
+  accentColorBox: {
+    marginRight: offset,
+    backgroundColor: Colors.accent,
+    width: 8,
+  },
+})
+
+export const InfoTextBox: React.FC<TextBoxProps> = ({ children }) => {
```
Comment (amanji) [inline App/components/texts/TextBox.tsx]: This is fine for now but we should try to keep one component per file. If needed we can extend from a BaseTextBox if the same styling is being applied for both

**Review by amanji — CHANGES_REQUESTED** [review]

File: App/components/buttons/Button.tsx (hunk 3 above)
Comment (jleach) [reply App/components/buttons/Button.tsx:55]: I'd like to remove those props in favour of `ButtonType` so until we run through and shore up the theme I'd like to leave em for now.

File: App/components/buttons/Button.tsx (hunk 4 above)
Comment (jleach) [reply App/components/buttons/Button.tsx:65]: I'd like to remove those props in favour of ButtonType so until we run through and shore up the theme I'd like to leave em for now.

File: App/Theme.ts (hunk 2 above)
Comment (jleach) [reply App/Theme.ts]: Will do, you looking for `BaseColors.lightBlue`?

File: App/components/texts/TextBox.tsx (hunk 5 above)
Comment (jleach) [reply App/components/texts/TextBox.tsx]: Fixed.

File: App/Theme.ts (hunk 2 above)
Comment (jleach) [reply App/Theme.ts]: Fixed.

**Review by amanji — APPROVED** [review]

**Pull request merged**